frontend/components/MainPage/sections/GasBalanceSection.tsx

```diff
@@ -100,12 +96,21 @@ const TooltipContent = styled.div`
   }
 `;
 
-type GasBalanceSectionProps = { serviceConfigId: string };
```

Comment:
remove this from here as its only used deeper in `BalanceStatus` but lmk if there was another plan for it